content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,290 @@
+* [Non-graceful node shutdown](https://github.com/kubernetes/enhancements/issues/2268)
+* [Support 3rd party device monitoring plugins](https://github.com/kubernetes/enhancements/issues/606)
+* [Auth API to get self-user attributes](https://github.com/kubernetes/enhancements/issues/3325)
+* [Proxy Terminating Endpoints](https://github.com/kubernetes/enhancements/issues/1669)
+* [Expanded DNS Configuration](https://github.com/kubernetes/enhancements/issues/2595)
+* [Cleaning up IPTables Chain Ownership](https://github.com/kubernetes/enhancements/issues/3178)
+* [Minimizing iptables-restore input size](https://github.com/kubernetes/enhancements/issues/3453)
+* [Graduate the kubelet pod resources endpoint to GA](https://github.com/kubernetes/enhancements/issues/3743)
+* [Extend podresources API to report allocatable resources](https://github.com/kubernetes/enhancements/issues/2403)
+* [Move EndpointSlice Reconciler into Staging](https://github.com/kubernetes/enhancements/issues/3685)
+
+### Deprecations And Removals
+
+This release saw several removals:
+
+* [Removal of CSI Migration for GCE PD](https://github.com/kubernetes/enhancements/issues/1488)
+
+## Release Notes
+
+The complete details of the Kubernetes v1.28 release are available in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md).
+
+## Availability
+
+Kubernetes v1.28 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.28.0). To get started with Kubernetes, you can run local Kubernetes clusters using [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), etc. You can also easily install v1.28 using [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+ 
+## Release Team
+
+Kubernetes is only possible with the support, commitment, and hard work of its community. Each release team is made up of dedicated community volunteers who work together to build the many pieces that make up the Kubernetes releases you rely on. This requires the specialized skills of people from all corners of our community, from the code itself to its documentation and project management.
+
+We would like to thank the entire release team for the hours spent hard at work to ensure we deliver a solid Kubernetes v1.28 release for our community.
+
+Special thanks to our Release Lead, Grace Nguyen, for guiding us through a smooth and successful release cycle and facilitating the communication and collaboration between the different Release Teams taking advantage of her experience as a Shadow and Lead in these teams.
+
+## Ecosystem Updates
+
+* KubeCon + CloudNativeCon North America 2023 will take place in Chicago, Illinois, The United States of America, from 6 – 9 November 2023! You can find more information about the conference and registration on the [event site](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/).
```

Comment:
Could we mention that Kubecon China will be held on 26 – 28 SEPTEMBER, Shanghai?
-  https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/

Reply:
Thanks totally forgot about that. I have added it.